.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+# For anything not explicitly taken by someone else:
+*                 @bogdandrutu
+
+# Owners per module:
+builds/*          @rghetia
```

Comment:
I think you want a leading slash. As-is, this rule is "a subdirectory called builds anywhere in the repo"